test/js-native-api/test_object/test_object.c

```diff
@@ -30,6 +30,39 @@ static napi_value Get(napi_env env, napi_callback_info info) {
   return output;
 }
 
+static napi_value GetNamed(napi_env env, napi_callback_info info) {
+  size_t argc = 2;
+  napi_value args[2];
+  char key[256] = "";
+  size_t key_length;
+  NAPI_CALL(env, napi_get_cb_info(env, info, &argc, args, NULL, NULL));
+
+  NAPI_ASSERT(env, argc >= 2, "Wrong number of arguments");
+
+  napi_valuetype valuetype0;
```

Comment:
Nit: Perhaps it would be more consistent to name the NAPI variables like `value_type0` instead of `valuetype0`, like is done for other variables like `key_length` above. Or is this intentionally done for the NAPI types?